src/main.go

```diff
@@ -61,6 +61,10 @@ func main() {
 		// can be quite flakey. For this reason, we wrap most issues in a
 		// non-fatal error type.
 		if pluginConfig.FailBuildOnPluginFailure || runtimeerrors.IsFatal(err) {
+			// If timeout error, exit 75 (POSIX timer expired)
```

Comment:
Another possibility would be for us to add a `FatalError` interface with an `ExitCode()` function in the `runtimeerrors` package.

I'm not sure whether it's better for the error to specify its exit code or for there to be a growing number of this kind of predicate in `main`.

Let's start here and see where it ends up.

In Culture Amp's use of the plugin, we only fail if there is an error resulting from a threshold failure, so this doesn't affect us.